app/models/article_result.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ArticleResult
+  FORMAT_TO_ICON = {
+    'Academic Journal' => 'book-open-4.svg',
+    'Books' => 'notebook.svg',
+    'News' => 'newspaper.svg',
+    'eBook' => 'notebook.svg',
+    'Video Recording' => 'camera-film-1.svg'
+  }.freeze
```

Comment:
```suggestion
  FORMAT_TO_ICON = {
    'Academic Journal' => 'book-open-4.svg',
    'Article' => 'wrap-text-around.svg',
    'Audio' => 'microphone-3.svg',
    'Books' => 'notebook.svg',
    'eBooks' => 'notebook.svg',
    'Maps' => 'map-pin-1.svg',
    'Music scores' => 'file-music-1.svg',
    'News' => 'newspaper.svg',
    'Report' => 'notepad-1.svg',
    'Video Recording' => 'camera-film-1.svg',
    'Videos' => 'camera-film-1.svg'
  }.freeze
```

and adding some of the actual articles+ formats.